plugin/trino-resource-group-managers/src/main/java/io/trino/plugin/resourcegroups/SelectorSpec.java

```diff
@@ -44,7 +44,7 @@ public SelectorSpec(
             @JsonProperty("originalUser") Optional<Pattern> originalUserRegex,
             @JsonProperty("authenticatedUser") Optional<Pattern> authenticatedUserRegex,
             @JsonProperty("source") Optional<Pattern> sourceRegex,
-            @JsonProperty("queryText") Optional<Pattern> queryText,
+            @JsonProperty("queryText") Optional<Pattern> queryTextRegex,
```

Comment:
Should we update `resource-groups.md`? 

https://github.com/trinodb/trino/blob/d854aece4e06c2076b2e6ed4fc3df7ae477dac6b/docs/src/main/sphinx/admin/resource-groups.md?plain=1#L184

https://github.com/trinodb/trino/blob/d854aece4e06c2076b2e6ed4fc3df7ae477dac6b/docs/src/main/sphinx/admin/resource-groups.md?plain=1#L256